README.md

```diff
@@ -22,16 +22,23 @@ CentOS Linux 8. It does not support CentOS Stream.
    stale repositories.
 1. Disable all non-CentOS repositories. You can re-enable the repos after the switch.
 1. Ensure you have at least 5GB of free space in `/var/cache`.
+1. Ensure at least 100GB of free space in /boot if it is a separate partition
 1. All automatic updates, e.g. via `yum-cron` should be disabled.
 
+**NOTES:** 
+
+1. You can use centos2ol_wrapper.sh script to automate checking of the space prerequisites for /var and /boot. It also creates some useful backups which might help you to recover from the situation when the script did not finishes successfully and you will have semi-converted system.
```

Comment:
The functionality of `centos2ol_wrapper.sh` should be incorporated into `centos2ol.sh`.